src/streaming/controllers/ProtectionController.js

```diff
@@ -439,20 +496,26 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this[MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_STATUSES_CHANGED] = onKeyStatusesChanged.bind(this);
 
             keySystems = this.protectionExt.getKeySystems();
-            this.protectionModel = this.system.getObject("protectionModel");
-            this.protectionModel.init();
+
+            // For browsers that don't support any version of EME, the app crashes without this check
+            if (this.system.hasMapping("protectionModel")) {
+                this.protectionModel = this.system.getObject("protectionModel");
+                this.protectionModel.init();
+            }
 
```

Comment:
@greg80303 Currently the ProtectionController setup method is invoking the ProtectionModel instance, it may be too late for the check in MediaPlayer. We would need to revisit the construction of ProtectionModel too.